POS/Integration/06_CreateStore.md

```diff
@@ -170,26 +184,25 @@ The store wizard has been started, see [Start the store wizard](#start-the-store
 
 ![Wizard 05](../../Assets/Screenshots/POS/Management/Stores/StoreWizard/SW05.png "[Store Wizard]")
 
-1. Select the warehouse you want to take the stock from in the *Warehouse* drop-down list.
+1. Click the *Warehouse* drop-down list and select the warehouse you want to take the stock from.
 
 2. Activate the *Automatically assign product to warehouse when activated in store* toggle to automate the assignment process. It is recommended to activate this option.
 
-    > [Info] A product must have stock in a warehouse in order to be sold from that warehouse. Products only have stock in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding channels offer is activated.    
+    > [Info] A product must have a stock level in a warehouse in order to be sold from that warehouse. Products only have stock level in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding sales channels offer is activated. 
+
     This automation only works if you have enabled the storage of multiple items per shelf when creating the warehouse. For detailed information, see [Configure the warehouse for POS](./01_ConfigureWarehouse.md).
 
 3. Enter a shelf number for the products in the *Shelf* field. The shelf number can be any arbitrary number.
 
-    > [Info] A product must have stock in a warehouse in order to be sold from that warehouse. Products only have stock in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf.
-
 4. Activate the *Configure stock withdrawal matrix* toggle to automate the configuration of the stock withdrawal matrix. It is recommended to activate this option.
```

Comment:
Stock withdrawal oder stock allocation? 

Stock allocation = Berechnung des Lagesbestand für Sales channel (POS in diesem Fall)
Stock withdrawal matrix = Entscheidung von welchem Lager die Materialien entnommen werden